test/versioned/undici/requests.test.js

```diff
@@ -426,3 +426,35 @@ test('Undici request tests', async (t) => {
     })
   })
 })
+
+test('Undici error reporting tests', async (t) => {
```

Comment:
ok i was hoping to just approve this and make follows up but this test isn't working correctly. I'll post a diff of what should be applied to this PR for approval.

Reply:
Can you make these changes @Voziv? https://github.com/newrelic/node-newrelic/commit/0b61de53831f1dee6221468aedaba72ce102576e. The test after doesn't seem to re-registering the undici hooks, so just adding a test inline above will work.  Also this adds documentation to the feature_flag.md

Reply:
Yep for sure! I've pushed those changes up